src/main/resources/tags/info/bungee.tag

```diff
@@ -3,6 +3,10 @@ aliases: bungeecord, velocity
 
 ---
 
-The Geyser and Floodgate BungeeCord plugins only need to be installed on the BungeeCord proxy unless you intend to use the Floodgate API. That way, you need Floodgate on the "backend" servers too.
-Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
-Run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.
+For the most basic installation, the Geyser and Floodgate Bungeecord or Velocity plugins only need to be installed on the proxy.
+
+For more advanced/complete features, you may also install Floodgate on the backend servers:
+***-*** It makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying (once they're uploaded to the skin server).
+***-*** If you intend to use certain plugins such as [BedrockPlayerManager](https://www.spigotmc.org/resources/bedrock-player-managment-for-floodgate-geyser.82278/)
+***-*** If you wish to use the API for your own purposes. You may run `!!api` in <#613194762249437245> for more information regarding the Floodgate API.
```

Comment:
```suggestion
***-*** If you intend to use plugins that require the Floodgate API on backend servers, such as [BedrockPlayerManager](https://www.spigotmc.org/resources/bedrock-player-managment-for-floodgate-geyser.82278/).
***-*** If you wish to use the Floodgate API for your own purposes. You may run `!!api` in <#613194762249437245> for more information.
```

This is a bit more clear for the end user imo

Reply:
TOH-MAY-TOH/TOH-MAH-TOH really. Saying the same thing, just worded differently.